.github/workflows/bump-api-schema-sha.yml

```diff
@@ -26,7 +25,8 @@ jobs:
 
           filepath="src/gatsby/utils/resolveOpenAPI.ts"
           sha="$(curl -sSL 'https://api.github.com/repos/getsentry/sentry-api-schema/commits/main' | awk 'BEGIN { RS=",|:{\n"; FS="\""; } $2 == "sha" { print $4 }')"
-          sed -i -e 's|^const SENTRY_API_SCHEMA_SHA =.*$|const SENTRY_API_SCHEMA_SHA = "'$sha'"|g' "$filepath"
+          sed -i -e "s|^const SENTRY_API_SCHEMA_SHA =.*$|const SENTRY_API_SCHEMA_SHA = '$sha';|g" "$filepath"
+          yarn prettier --write "$filepath"
```

Comment:
Should this run eslint --fix instead?

Reply:
I think they do the same thing for this scenario? `eslint --fix` is fine though if that's preferred

Reply:
`eslint --fix` should fix more issues